docs/resources/dms_kafka_user.md

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Distributed Message Service (DMS)"
+---
+
+# huaweicloud_dms_kafka_user
+
+Manages a DMS kafka user resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "kafka_instance_id" {}
+
+resource "huaweicloud_dms_kafka_user" "user" {
+  instance_id = var.kafka_instance_id
+  name        = "user_1"
+  password    = "Test@123"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the DMS kafka user resource. If omitted, the
+  provider-level region will be used. Changing this creates a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of the DMS kafka instance to which the user belongs.
+  Changing this creates a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the user. Changing this creates a new resource.
+
+* `password` - (Required, String) Specifies the password of the user. The parameter must be 8 to 32 characters
+  long and contain only letters(case-sensitive), digits, and special characters(`~!@#$%^&*()-_=+|[{}]:'",<.>/?).
```

Comment:
add validation to schema?

Reply:
the rule is vary loose, so validation is not necessary